### ISSUE TITLE
Legion Plugin logging error on second build

### ISSUE THREAD

**Issue by MaksymBilenko** [legion-platform/legion#301]
Issue with same log file
```+ cp script-log.txt /home/jenkins/workspace/Legion-demo
cp: 'script-log.txt' and '/home/jenkins/workspace/Legion-demo/script-log.txt' are the same file```

**Comment by akharlamov** [comment]
Guys, 

I do not think this is fixed just yet. We have a dirty hack to make it going. But a proper resolution should be implemented. 

Cheers

-- Alexey

**Comment by kirillmakhonin** [comment]
@akharlamov @MaksymBilenko  as I understand, this problem have appeared only when you'd placed Jenkinsfile in the root of repository, right?

**Comment by kirillmakhonin** [comment]
It looks like you have tried to train and deploy model from Jenkinsfile that has been placed in the repository root. So, in this case current code works incorrectly and tries to copy log files from one directory to same directory. That is not linked with build number.

**Comment by kirillmakhonin** [comment]
Closed by #388
